src/compiler/scala/tools/nsc/settings/ScalaSettings.scala

```diff
@@ -507,8 +507,7 @@ trait ScalaSettings extends StandardScalaSettings with Warnings { _: MutableSett
     .withAbbreviation("-Yhot-statistics")
   val Yshowsyms       = BooleanSetting("-Vsymbols", "Print the AST symbol hierarchy after each phase.") withAbbreviation "-Yshow-syms"
   val Ytyperdebug        = BooleanSetting("-Vtyper", "Trace type assignments.") withAbbreviation "-Ytyper-debug"
-  val XlogImplicits      = BooleanSetting("-Vimplicits", "Show more detail on why some implicits are not applicable.")
-    .withAbbreviation("-Xlog-implicits")
+  val XlogImplicits      = BooleanSetting("-Xlog-implicits", "Show more detail on why some implicits are not applicable.")
```

Comment:
Should be removed.  Are there any usages left?  Or are they all replaced with `VexplainImplicits` usages?  Which, btw, should also be renamed so the val name matches the setting name.

Reply:
are we sure it shouldn't stay in as `Xlog-implicits` only?

Reply:
Yes, it shouldn't stay, with the new implementation replacing it.

Reply:
ok, then I'll remove the related tests!

Reply:
looks like the tests can remain, just had to remove parts of the error messages

Reply:
I think we want to keep those error messages, just update them with the new implementation.  I.e. switch them to `-Vimplicits`.

Reply:
only one of those tests has different output with `-Vimplicits` enabled, I changed it accordingly.